tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -83,6 +83,51 @@ Render modes
 | **fog_disabled**              | Disable receiving depth-based or volumetric fog. Useful for blend_add materials like particles.      |
 +-------------------------------+------------------------------------------------------------------------------------------------------+
 
+Coordinate spaces
```

Comment:
We should either put this section here, or at the bottom of the page. 
Pros for here:
- It's a prerequisite for understanding everything else on the page.
Pros for end of page:
- Once you already know what the spaces are, it may be annoying to scroll past it every time you're reading. The page already functions fine without a coordinate space section.

Reply:
Upon further reflection, this section should go at the bottom of the page unless there's a strong argument otherwise.